app/services/phone_formatter.rb

```diff
@@ -1,7 +1,5 @@
 module PhoneFormatter
-  DEFAULT_COUNTRY = 'US'.freeze
-
   def self.format(phone, country_code: nil)
```

Comment:
Can we make `country_code` a required param? To help highlight cases that assumed US

Reply:
I think the problem with that is that those places don't necessarily assume U.S., they just don't have access to a separate detail of what the country is expected to be. For example, in the [form code](https://github.com/18F/identity-idp/blob/e28310c64d5589a4691de917d51bbc4fa5e1cd80/app/forms/otp_delivery_selection_form.rb#L12) mentioned in the original comment, the phone number may not necessarily be a U.S. based number, and we don't want to assume that it is, because that'd lead to the same issue which prompted this change.

Phonelib seems to be reasonably capable of detecting the country automatically, or at least in the case that it isn't able to, it'll at least still fall back to some reasonable output (such as what we've been doing with trying to forcefully parse Canadian area codes as U.S. numbers).